src/internal/analytics/components/analytics-funnel.tsx

```diff
@@ -157,6 +157,7 @@ const InnerAnalyticsFunnel = ({ mounted = true, children, stepConfiguration, ...
         },
       ];
 
+      const componentTheme = THEME === 'polaris' && isVisualRefresh ? 'vr' : THEME;
```

Comment:
We have a similar piece of logic here: https://github.com/cloudscape-design/components/blob/7ff86dc1b6363ac4d16139f07590bd944799e019/src/internal/hooks/use-telemetry/index.ts#L9

Can we somehow deduplicate these, so we will not have conflicting data in different metrics?

Reply:
Some more fun places: https://github.com/cloudscape-design/board-components/blob/a364adaceaca5f794ce4500c2b792c834f649971/src/internal/base-component/use-telemetry.ts#L10

Reply:
Maybe a `useTheme(theme:string)` function in component-toolkit is needed